### ISSUE TITLE
Calling AppWindow apis does not work anymore for unpackaged apps

### ISSUE THREAD

**Issue by lhak** [microsoft/WindowsAppSDK#1815]
### Describe the bug

After updating from preview 3 to the stable 1.0 release, calling appwindow apis like AppWindow.GetFromWindowId() causes the app to crash:

SEHException: External component has thrown an exception.

The debug output also shows this message:

"An attempt to delay load a .dll or get a function address in a delay-loaded .dll file failed"

The same code works fine when running as a packaged application. I could reproduce this issue on multiple machines.

### Steps to reproduce the bug

see above

### Expected behavior

No crash

### NuGet package version

1.0.0

### Packaging type

Unpackaged

### Windows version

Windows 11 (22000)

### IDE

Visual Studio 2022

**Comment by qmatteoq** [comment]
I'll add more details since I'm getting the same error. I'm starting from a WPF sample app, in which I successfully integrated Windows Apps SDK Preview 2 and 3. This is the code I use to retrieve an AppWindow from a HWND:

```
var hWnd = new WindowInteropHelper(this).EnsureHandle();
WindowId myWndId = Microsoft.UI.Win32Interop.GetWindowIdFromWindow(hWnd);
appWindow = AppWindow.GetFromWindowId(myWndId);
```
After migrating to 1.0 final, the last line (AppWindow.GetFromWindowId()) returns the following exception:

```
System.Reflection.TargetInvocationException: 'Exception has been thrown by the target of an invocation.'
SEHException: External component has thrown an exception.
```

**Comment by andrewleader** [comment]
@rkarman @mevey it sounds like the AppWindow APIs aren't working from unpackaged WPF apps, even though it used to work in Preview 3?

**Comment by blake-fm** [comment]
Following the [first app tutorial](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app?pivots=winui3-unpackaged-csharp), adding the below to MainWindow.xaml.cs, repro's for me.

```
var hwnd = WinRT.Interop.WindowNative.GetWindowHandle(this);
var windowId = Win32Interop.GetWindowIdFromWindow(hwnd);
var appWindow = AppWindow.GetFromWindowId(windowId);

**Comment by rkarman** [comment]
I've filed a bug in our internal issue tracking system and assigned it to the dev team to investigate. I wouldn't expect any traction on this until after Thanksgiving though. I'll follow-up on Monday to make sure it is triaged gets eyes on it.

**Comment by blake-fm** [comment]
Adding supportedOS to appmanifest resolved this for me.
```
    <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
        <application>
            <!-- Windows 10 and Windows 11 -->
            <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}"/>
        </application>
    </compatibility>
```
https://docs.microsoft.com/en-us/windows/win32/sysinfo/targeting-your-application-at-windows-8-1

**Comment by aepot** [comment]
@blake-fm Workaround helped, thanks.

I also figured out that later removing these lines from the manifest makes no effect. The app continue working fine (Unpackaged, x64).
Looks like a kind of unpredictable behavior. 

Then I don't think that simple editing the manifest looks like a proper solution.

**Comment by blake-fm** [comment]
@aepot Do you get the same results on a rebuild / clean+build?  With app.manifest not being monitored by the FastUpToDate system, it might have just assumed it already had the most recent app.manifest settings?

Completely agree though - definitely not a solution, just a workaround~

**Comment by aepot** [comment]
@blake-fm sorry, really Rebuild did the job. Deleted the post not to mislead others. Didn't know that you already read it. :)

**Comment by lhak** [comment]
I can confirm that the workaround posted by @blake-fm works.

**Comment by DrusTheAxe** [comment]
See [#1891 for more ](https://github.com/microsoft/WindowsAppSDK/discussions/1891#discussioncomment-1948651)

The root problem there is Microsoft.UI.Windowing.Core.dll imports `ext-ms-win-windowing-internal-l1-1-0.dll!Windowing_DllGetActivationFactory` which doesn't exist downlevel (not on 21H1, I didn't hunt around to see when it first appeared).

**Comment by lhak** [comment]
I think this is fixed in 1.0.1?

**Comment by rkarman** [comment]
Yes. The fix is in 1.1 and it has been serviced to 1.0.1 as well.
